app/controllers/api/v1/court_hearings_controller.rb

```diff
@@ -41,7 +41,9 @@ def move
       end
 
       def should_save_in_nomis?
-        move && params['should_save_in_nomis'] == 'true'
+        do_not_save_to_nomis = params['do_not_save_to_nomis'] == 'true'
```

Comment:
Will this also support just passing the query param without a value?

It would be nice to have `/court_hearings?do_not_save_to_nomis` and `/court_hearings?do_not_save_to_nomis=true` behave the same.

No biggie if not

Reply:
Only 'true' will change the behaviour. Everything else is noop